src/App.tsx

```diff
@@ -1,156 +1,76 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { ShowError } from './components/ShowError/ShowError';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList/TodoList';
+import { Header } from './components/Header/Header';
+import { FilterBy } from './types/Filter';
 
-const USER_ID = 0;
+const USER_ID = 6156;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [todoFilter, setTodoFilter] = useState('All');
+  const [error] = useState('');
+
+  const isAnyTodoCompleted
+    = Boolean(todos.filter(todo => todo.completed).length);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(response => {
+        setTodos(response);
+        setVisibleTodos(response);
+      });
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const handleFilterClick = (filter: FilterBy) => {
```

Comment:
you don't need the word `handle` here. The names should be meaningful, I have to understand what's happening in the function body from its name.
```suggestion
  const changeTodosFilter = (filter: FilterBy) => {
```